src/rest/api.ts

```diff
@@ -32,3 +34,7 @@ export const useFetchUnderOppfolging = (fnr: string) =>
 	useFetch<UnderOppfolgingData>(`/veilarboppfolging/api/underoppfolging?fnr=${fnr}`);
 
 export const useFetchPersonalia = (fnr: string) => useFetch<PersonaliaInfo>(`/veilarbperson/api/person/${fnr}`);
+
+export const useFetchPersonaliaFraPdl = (fnr: string) => useFetch<PersonaliaV2Info>(`/veilarbperson/api/v2/person/${fnr}`);
+
+export const useFetchFeatureToggle = (toggle: string) => useFetch<Features>(`/veilarbpersonflatefs/api/feature?${toggle}`);
```

Comment:
Query param for feature til `useFetchFeatureToggle` er ikke helt riktig. Tenker også at det greit at den fetcher alle toggles, istedenfor 1 og 1, selv om det i dette tilfellet kun er 1 toggle som skal hentes